### PR TITLE
responsive navbar

### PR DESCRIPTION
**Pull request by theprojectcode**
**Navbar is responsive now. Photos below show the navbar menu becoming responsive at smaller breakpoints.   ** 
Let me know if how my code looks. Would love feedback

Screen Shot 2021-11-04 at 7 20 28 PM

Screen Shot 2021-11-04 at 7 20 40 PM

### CODE REVIEW

**Comment by theprojectcode** [comment]
I realize that both of my build test failed. It looks like it failed because of the aioutline icon. Not sure why? I have it as a package.

**Comment by Neumand** [comment]
@theprojectcode 

Maybe try running `npm i react-icons`? I don't see it in the package.json. Or you could use an icon from the `@chakraui/icons` package as I see that's already installed!

**Comment by with-heart** [comment]
Let's go with `react-icons` for the time being! That gives us access to a much larger set of icons than `@chakra-ui/icons`.

Also welcome to open source @theprojectcode! ❤️ 

@allcontributors please add @theprojectcode for code

**Comment by with-heart** [comment]
Also since we're using `yarn` in this project, be sure to use `yarn add react-icons` :)

File: src/components/Nav/index.tsx [hunk 1]
```diff
@@ -1,66 +1,137 @@
...
+              title="Choc Home Page"
+              display="flex"
+              alignItems="center"
+            >
+              <Logo h={7} w={7} />
+            </chakra.a>
+            <chakra.h1 fontSize="m" fontWeight="medium" ml="2">
+              Developer DAO
+            </chakra.h1>
+          </Flex>
```
Comment (with-heart) [inline src/components/Nav/index.tsx:47]: Changing the way we used `Link` here made it so "Developer DAO" isn't part of the link, so we can't click it to return to the homepage. Only the logo works as a link.

We can safely revert this part to what it was previously and it'll still work fine.

```tsx
        <Link href="/" passHref>
          <HStack as="a" title={t('title')} display="flex" alignItems="center">
            <Logo h={7} w={7} />
            <chakra.span
              fontWeight="bold"
              fontSize="sm"
              color="gray.600"
              transition="color 300ms ease-in-out"
              _hover={{ color: 'black' }}
            >
              {t('title')}
            </chakra.span>
          </HStack>
        </Link>
```

File: src/components/Nav/index.tsx [hunk 2]
```diff
@@ -1,66 +1,137 @@
...
+              <Link href="/projects" passHref>
+                {t('projects')}
+              </Link>
+              <IconGitHub
+                h={7}
+                w={7}
+                opacity={0.6}
+                transition="opacity 300ms ease-in-out"
+                _hover={{ opacity: 1 }}
+              />
```
Comment (with-heart) [inline src/components/Nav/index.tsx:70]: Looks like we missed the part of the code that makes this a link! 😆 

```suggestion
            <chakra.a
              href="https://github.com/Developer-DAO/developer-dao"
              target="_blank"
              rel="noreferrer"
              title={t('daoGithubRepo')}
            >
              <IconGitHub
                h={7}
                w={7}
                opacity={0.6}
                transition="opacity 300ms ease-in-out"
                _hover={{ opacity: 1 }}
              />
            </chakra.a>
```

File: src/components/Nav/index.tsx [hunk 3]
```diff
@@ -1,66 +1,137 @@
...
+
+              <VStack
+                pos="absolute"
+                top={2}
+                left={0}
+                right={0}
+                display={mobileNav.isOpen ? 'flex' : 'none'}
+                flexDirection="column"
+                pt={7}
+                pb={7}
```
Comment (with-heart) [inline src/components/Nav/index.tsx:92]: Just fyi! Chakra has a shortcut syntax that handles `pt` and `pb` at the same time: `py`

So `py={7}`. Not worth changing, just something to be aware of!

There's also `px` that handles `pl`+`pr`

File: src/components/Nav/index.tsx [hunk 4]
```diff
@@ -1,66 +1,137 @@
...
+            </chakra.a>
+            <chakra.h1 fontSize="m" fontWeight="medium" ml="2">
+              Developer DAO
+            </chakra.h1>
+          </Flex>
+          <HStack display="flex" alignItems="center" spacing={1}>
+            <HStack
+              spacing={5}
+              mr={1}
+              color="brand.500"
```
Comment (with-heart) [inline src/components/Nav/index.tsx:52]: We should replace these props with the original:

```suggestion
              spacing={{ base: 3, sm: 10 }}
```

**Review by with-heart — CHANGES_REQUESTED** [review]
This is a great first contribution! I noticed some differences between the original design and this new one though, so I left a few comments explaining what changed and how we can fix it. 

Happy to pair and work through the requested changes if need be!

**Comment by theprojectcode** [comment]
> This is a great first contribution! I noticed some differences between the original design and this new one though, so I left a few comments explaining what changed and how we can fix it.
> 
> Happy to pair and work through the requested changes if need be!

Hey man, thanks for the feedback. I'd love to pair and work through those things as well. Let's set up a time.

**Comment by theprojectcode** [comment]
> > This is a great first contribution! I noticed some differences between the original design and this new one though, so I left a few comments explaining what changed and how we can fix it.
> > Happy to pair and work through the requested changes if need be!
> 
> Hey man, thanks for the feedback. I'd love to pair and work through those things as well. Let's set up a time.

@with-heart

**Review by with-heart — APPROVED** [review]

**Review by narbs91 — APPROVED** [review]
LGTM!  Awesome stuff gang love the collaboration/team-work that happened here!  Congrats @theprojectcode for grinding through the trials of learning React, well done!

**Comment by theprojectcode** [comment]
Thank you! @narbs91 I couldn't have done it without @with-heart thank you guys for your support!

**Pull request merged**